11/archlinux/Dockerfile

```
@@ -20,7 +20,22 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
-FROM archlinux:latest AS archlinux
+FROM archlinux:latest AS jre-build
```

This is intentionally not coming from Eclipse Temurin?
```suggestion
FROM archlinux:latest AS jre-build
```

---

I didn’t bother no, I question whether this image should be here at all.

I wasn’t sure if there was implications from getting it from a different OS type like Debian whether libraries would be different…